CMakeLists.txt

```diff
@@ -1449,15 +1449,8 @@ endif()
 # Installation preparation.
 #
 
-if(WIN32 AND NOT CYGWIN)
-  set(DEF_INSTALL_CMAKE_DIR cmake)
-else()
-  set(DEF_INSTALL_CMAKE_DIR lib/cmake/libevent)
-endif()
-
 set(EVENT_INSTALL_CMAKE_DIR
-    "${CMAKE_INSTALL_PREFIX}/${DEF_INSTALL_CMAKE_DIR}"
-    CACHE PATH "Installation directory for CMake files")
+    "${CMAKE_INSTALL_PREFIX}/lib/cmake/libevent")
```

Comment:
So we cannot change this from the cmake args - ok

Reply:
Consider these steps:
```
cmake -DCMAKE_INSTALL_PREFIX=/first/install/path ..
cmake -DCMAKE_INSTALL_PREFIX=/second/install/path ..
cmake --build . --target install
```
The final products are always installed to /first/install/path because `EVENT_INSTALL_CMAKE_DIR` was cached. So I removed the `Cache` key.